packages/rrweb/src/record/observer.ts

```diff
@@ -753,6 +755,45 @@ function initFontObserver({ fontCb, doc }: observerParam): listenerHandler {
   };
 }
 
+function initSelectionObserver(param: observerParam): listenerHandler {
+  const { mirror, blockClass, selectionCb } = param;
+  let collapsed = true;
+
+  const updateSelection = () => {
+    const selection = document.getSelection();
```

Comment:
```suggestion
    const selection = doc.getSelection();
```
If the website has an iframe element, the document has to be iframe's document and it can be retrieved from the param.

Reply:
yeah. this would be more appropriate

Reply:
Got it, thanks for your explanation

Reply:
@0jinxing I guess you may miss this one